src/derivative_wrappers.jl

```diff
@@ -1,6 +1,6 @@
-struct SensitivityAlg{CS,AD,FDT} <: DiffEqBase.DEAlgorithm end
-Base.@pure function SensitivityAlg(;chunk_size=0,autodiff=true,diff_type=Val{:central})
-  SensitivityAlg{chunk_size,autodiff,typeof(diff_type)}()
+struct SensitivityAlg{CS,AD,FDT,Jv} <: DiffEqBase.DEAlgorithm end
+Base.@pure function SensitivityAlg(;chunk_size=0,autodiff=true,diff_type=Val{:central},autojacvec=false)
```

Comment:
I think we should default to true here? I think the timings will show you almost always want to do this.